gemrb/plugins/CHUImporter/CHUImporter.cpp

```diff
@@ -377,7 +377,7 @@ Window* CHUImporter::GetWindow(ScriptingId wid) const
 				TextArea* ta = new TextArea( ctrlFrame, fnt, ini, fore, init, back );
 
 				if (SBID != 0xffff) {
-					ScrollBar* sb = GetControl<ScrollBar>(SBID, win);
+					ScrollBar* sb = dynamic_cast<ScrollBar*>(GetControl(SBID, win));
```

Comment:
odd that Apple's version of clang (based from 3.7 I think) doesn't take issue with this. Anyhow I will merge this as is.